parachain/test-parachains/adder/collator/src/cli.rs

```diff
@@ -45,6 +45,10 @@ pub struct RunCmd {
 	#[allow(missing_docs)]
 	#[structopt(flatten)]
 	pub base: sc_cli::RunCmd,
+
+	/// Id of the parachain this collator collates for.
+	#[structopt(long)]
+	pub parachain_id: Option<u32>,
```

Comment:
alternatively, this could be `u32` with the `default_value = "100"`.

Reply:
yeah, I wanted to be consistent [with cumulus](https://github.com/paritytech/cumulus/blob/2662bcd3dda634bfafb9632da8fa1fe85a2d71d4/rococo-parachains/src/cli.rs#L98-L98). But now I've noticed [this](https://github.com/paritytech/cumulus/blob/2662bcd3dda634bfafb9632da8fa1fe85a2d71d4/rococo-parachains/src/cli.rs#L64-L64) xD

if you don't mind I prefer to keep it this way to just not suffer another cycle of CI

Reply:
I definitely don't mind :)